### ISSUE TITLE
Support for node removal

### ISSUE THREAD

**Issue by Glandos** [mwri/cytoscape-dom-node#5]
Hi, currently, this (nice) plugin only watch for node adding, but when a node is removed, the HTML DOM node is not removed, leading to… garbage display.

Is it plan to handle node removal?

**Comment by mwri** [comment]
Ah, good shout. Should be pretty trivial to fix. I guess there are two cases that should be handled, the data changes so there is no longer a 'dom' value, or the cytoscape node is actually removed from the graph.

**Comment by nielsboogaard** [comment]
I've implemented this in my fork, see https://github.com/mwri/cytoscape-dom-node/compare/master...nielsboogaard:cytoscape-dom-node:master. 
I can create a PR for it if you want @mwri

**Comment by nielsboogaard** [comment]
> Hi @nielsboogaard I think it would be great if you did a PR. Looking at your diff, shouldn't it listen to the `remove` event though to facilitate this?

Yes, that's what I said. Implemented it now upon changes in the data object, but can be extended with listening to remove as well. Will do and create a PR after that.

**Comment by bradley-edward** [comment]
> > Hi @nielsboogaard I think it would be great if you did a PR. Looking at your diff, shouldn't it listen to the `remove` event though to facilitate this?
> 
> Yes, that's what I said. Implemented it now upon changes in the data object, but can be extended with listening to remove as well. Will do and create a PR after that.

Hey. Did you managed to get the PR done and ready to submit?

**Comment by nielsboogaard** [comment]
> > > Hi @nielsboogaard I think it would be great if you did a PR. Looking at your diff, shouldn't it listen to the `remove` event though to facilitate this?
> > 
> > 
> > Yes, that's what I said. Implemented it now upon changes in the data object, but can be extended with listening to remove as well. Will do and create a PR after that.
> 
> Hey. Did you managed to get the PR done and ready to submit?

Not yet, very busy lately and the coming weeks. Hopefully more time in some weeks.
